pkg/node/node.go

```diff
@@ -988,13 +1009,26 @@ func (s *Service) NodeGetInfo(ctx context.Context, req *csi.NodeGetInfoRequest)
 					continue
 				}
 
-				_, err = s.iscsiLib.DiscoverTargets(infoList[0].Portal, false)
-				if err != nil {
-					log.Error("couldn't discover targets")
-					continue
-				}
+				if !s.useFC && s.useISCSI {
```

Comment:
for better readability, please add a "else if s.useISCSI" in line 1005 and else case for nvme in 1032. Basically like - if fc, else if iscsi and else.

Reply:
Since the network IP for both iSCSI and NVMe is same, the GetISCSITargetsInfoFromStorage() function used in line 1006 is used for both iSCSI and NVMe.
Therefore if FC is not present, then either iSCSI or NVMe will be present, and so we are using the function (for ISCSI or NVMe)